src/main/java/graphql/schema/idl/SchemaTypeChecker.java

```diff
@@ -90,7 +85,7 @@
 
     private void checkForMissingTypes(List<GraphQLError> errors, TypeDefinitionRegistry typeRegistry) {
         // type extensions
-        List<ObjectTypeExtensionDefinition> typeExtensions = typeRegistry.objectTypeExtensions().values().stream().flatMap(Collection::stream).collect(Collectors.toList());
+        List<ObjectTypeExtensionDefinition> typeExtensions = typeRegistry.objectTypeExtensions().values().stream().flatMap(Collection::stream).collect(toList());
```

Comment:
please use the util function

Reply:
`FpKit#flatList` applies to `List<List<>>` and we have a `Collection<List<>>` here. 
Should we create a more generic flatten method in `FpKit`?